projects/igniteui-angular/src/lib/date-picker/date-picker.component.ts

```diff
@@ -790,6 +790,8 @@ export class IgxDatePickerComponent extends PickerBaseDirective implements Contr
                 this._ngControl.statusChanges.subscribe(this.onStatusChanged.bind(this));
             if (this._ngControl.control.validator) {
                 this.inputGroup.isRequired = this.required;
+                this._renderer.setAttribute(this.inputDirective.nativeElement, 'aria-required',
```

Comment:
Same comment as the combo here. What's stopping us from binding this attribute directly in the template?